src/translations/en.json

```diff
@@ -4181,7 +4181,7 @@
       "open_release_notes": "Open release notes",
       "create_backup": "Create backup before updating",
       "description": "There is an update available for the {name}. You have {version} installed. Click update to update to version {newest_version}",
```

Comment:
For another PR:

Just realized that this description is not correctly translatable for all languages. In English "the" works as pronoun for all names, but other languages can have gendered prononus and operating system, core, supervisor and add-on might have different pronouns.

Reply:
<https://github.com/home-assistant/frontend/pull/10669>